orion-ui/src/App.vue

```diff
@@ -79,9 +83,14 @@
 @screen lg {
   .app {
     --prefect-scroll-margin: theme('spacing.2');
-
     display: grid;
     grid-template-columns: max-content minmax(0, 1fr);
   }
 }
+
+.app__router-view {
+  /* The 1px flex-basis is important because it allows us to use height: 100% without additional flexing */
+  flex: 1 0 1px;
+  height: 100%;
```

Comment:
Won't this make this div scroll rather than the whole page?

Reply:
If I understand your meaning then no, this will cause the element to expand to 100% of the specified parent height which, with flex-basis set at 1px, will be the computed height of the cell after Flexbox calculations.

Reply:
Okay, I trust you lol.